src/utils/config.ts

```diff
@@ -19,7 +19,8 @@ const QUERY_CHUNK_SIZE = 10_000;
  */
 const CONFIG_DEFAULT: ConfigMap = Object.freeze({
     [CONFIG_KEY.DECODER_OPTIONS]: {
-        formatString: "%d{yyyy-MM-dd HH:mm:ss.SSS} [%process.thread.name] %log.level %message%n",
+        formatString: "{@timestamp:timestamp:YYYY-MM-DD HH\\:mm\\:ss.SSS}" +
+        " [{process\\.thread\\.name}] {log\\.level} {message}",
```

Comment:
Sorry for this legacy issue - let's put the log level directly after the timestamp so we can stay consistent with the YScope-recommended text log format string. (Why we recommend placing the log level directly after the date string is because IRv1 does not carry any authoritive log level info, and we have to parse the log level at a certain position as you know)
```suggestion
        formatString: "{@timestamp:timestamp:YYYY-MM-DD HH\\:mm\\:ss.SSS}" +
        " {log\\.level} [{process\\.thread\\.name}] {message}",
```